Week2/homework/js-exercises/Is-it-bigger-than-10.js

```diff
@@ -0,0 +1,16 @@
+function checkDoubleDigits(num) {
+    return new Promise ((resolve, reject) =>{
+        
+        setTimeout(() => {
+            num = Math.floor(Math.random() * 100)
+                if (num > 10){
+                resolve (`The number ${num} is bigger than 10!`)
+                } else {
+                    reject(`The number ${num} is samller than 10!`)
+                }
+            }, 1000)
+        })
+    }
+checkDoubleDigits()
+.then (res => console.log(res))
+.catch (err => console.log(err))
```

Comment:
```suggestion
checkDoubleDigits()
.then (console.log)
.catch (console.log)
```
I am just showing off 👅